gnomad_constraint/utils/constraint.py

```diff
@@ -66,17 +67,25 @@ def add_vep_context_annotations(
         Function expects that multiallelic variants in the VEP context Table have been
         split.
 
+    Function also adds an_strata_sample_count to globals.
```

Comment:
```suggestion
    Function also adds 'an_strata_sample_count' to globals.
```

Reply:
Probably need to change this so support cases where that annotation is not on the context HT